src/System.Text.Utf8/src/System/Text/Utf8/Utf8CodeUnit.cs

```diff
@@ -2,21 +2,26 @@
 
 namespace System.Text.Utf8
 {
-    // TODO: Should we directly use byte or leave it. If we leave it should we leave the name?
-    public struct Utf8CodeUnit : IEquatable<Utf8CodeUnit>
+    // TODO: Check if name is understandable for users
+    //       I personally think it is better to keep the spec name than calling it with some random new name or byte
+    //       In the end it is less confusing than trying to figure out what spec means and what .NET means
+    public partial struct Utf8CodeUnit : IEquatable<Utf8CodeUnit>
     {
+        private byte _value;
```

Comment:
I couldn't see from the diff, why did you switch from the property to a private variable?